apps/frontend/src/components/PostReqCourses.tsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import Link from "next/link"; // Import Next.js Link component
+import { useFetchCourseInfo } from "~/app/api/course";
+
+interface TreeNode {
+  courseID: string;
+  postreqs?: TreeNode[];
+}
+
+interface Props {
+  courseID: string;
+}
+
+export const PostReqCourses = ({ courseID }: Props) => {
+  const { isPending: isCourseInfoPending, data: info } = useFetchCourseInfo(courseID);
+
+  if (isCourseInfoPending || !info) {
+    return null;
+  }
+
+  // Recursive function to render only the child branches
+  const renderTree = (nodes: TreeNode[]) => {
+    return (
+      <div style={{ display: "flex", flexDirection: "column" }}>
+        {nodes.map((node) => (
+          <div
+            key={node.courseID}
+            style={{
```

Comment:
We use [tailwindcss](https://tailwindcss.com) for styling. So instead of putting style={{}} in each component, we use className=“”. Could you change all the CSS styling to that format for consistency please?